package.json

```diff
@@ -185,6 +186,8 @@
     "easydl": "^1.0.3",
     "electron-store": "^8.0.1",
     "electron-updater": "^5.0.1",
+    "embla-carousel-autoplay": "^8.0.0-rc12",
+    "embla-carousel-react": "^8.0.0-rc12",
```

Comment:
What is this library for? I cannot see it being imported in the code anywhere.

Reply:
It is a peer dependency from the ui library. We cannot ship it as a dependency currently because it breaks nextjs. This is because mantine loses the isBrowser/isNode checks when being built in the ui library

When updating to the latest ui library version for elio's fix for the generic button on change typing, i also had to add these as it included the carousel refactor

Reply:
That's not really convenient. So on every place we need to use the UI we also need to install these dependencies? What about trying a different way of bundling the UI lib?

Reply:
yes. I have spent a decent amount of time on this and it is pretty complex/involved. Not worth the time at the moment in my opinion. we only use the ui library in a couple of repos